build/localnet/manifests/configs.yaml

```diff
@@ -36,6 +36,7 @@ data:
       },
       "p2p": {
         "port": 42069,
+        "hostname": "",
```

Comment:
### Rationale

![image](https://user-images.githubusercontent.com/600733/231687943-663728b3-12f6-4c6b-b04a-b767e241702c.png)

### Is there an issue for this?

Reply:
:thinking: Unfortunately I don't think it's possible to add a comment inline with the JSON as that would produce invalid JSON. Comments were introduced in [JSON5](https://json5.org/), which has been slow to catch on :sweat_smile: